nearswap/src/pool.rs

```diff
@@ -41,6 +42,16 @@ pub struct Pool {
     pub shares: LookupMap<AccountId, Balance>,
     /// check `PoolInfo.total_shares`
     pub total_shares: Balance,
+
+    populated: usize,
+    last_updated_index: usize,
+    pivoted: bool,
+    pub observation: Vec<Twap>,
```

Comment:
let's rename - should be plural (observations)

Reply:
ok